script.js

```diff
@@ -0,0 +1,25 @@
+function sum() {
+    var a = document.getElementById("a").value
+    var b = document.getElementById("b").value
```

Comment:
Naming is important, always name things what they are. Names like `a` and `b` don't provide any context to the reader.

Reply:
Oh man. You said that already!

Reply:
I totally forgot.